weatherman/weatherreport.py

```diff
@@ -0,0 +1,65 @@
+"""
+For reading data and generating weather report
+"""
+import os
+from datetime import datetime
+
+from weatheryearlysummary import WeatherYearlySummary
+
+
+class WeatherReport:
+    KEY_DATE = "PKT"
+    KEY_DATE_ALTER = "PKST"
+    KEYS_WEATHER_ATTRIB = ["Max TemperatureC", "Min TemperatureC", "Max Humidity", "Min Humidity", KEY_DATE]
+
+    def __init__(self, path_to_dir):
+        self.__path_to_dir = path_to_dir
+        self.weather_data = dict()
+
+    def __generate_header_index(self, header):
+        header_index = [header.index(attributes) for attributes in self.KEYS_WEATHER_ATTRIB[:-1]]
```

Comment:
Only using this here. if we don't need the last index lets remove from definition.